differential_pat.json

```diff
@@ -0,0 +1,506 @@
+{
+    "resourceType": "Bundle",
+    "meta" : {
+      "security" : [
+      {
+        "system" : "http://terminology.hl7.org/CodeSystem/v3-ObservationValue",
+        "code" : "SUBSETTED"
+      }
+    ]
+  },
```

Comment:
Besides the addition of this security tag this bundle looks like the updated bundle. The IG contains a [differential example](https://build.fhir.org/ig/HL7/davinci-pas/pa-request-example-homehealthcare-differential.json.html). I also assume this was only added for use in testing? It should be removed before merging to master and put into a postman collection